src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/zipfilestatus/ZipFileEnvelope.java

```diff
@@ -33,7 +47,11 @@ public ZipFileEnvelope(
         String ccdId,
         String envelopeCcdAction,
         boolean zipDeleted,
-        String rescanFor
+        String rescanFor,
+        List<ScannableItemResponse> scannableItems,
+        List<NonScannableItemResponse> nonScannableItems,
+        List<PaymentResponse> payments
```

Comment:
List<String> would be enough?

Reply:
This is the same as in `EnvelopeResponse`, used in `EnvelopeController` for `/envelopes/{envelopeId}' endpoint.